### PR TITLE
404 fixes mostly

### PR DESCRIPTION
**Pull request by photodow**
### Related Ticket(s)

https://github.ibm.com/webstandards/digital-design/issues/1671

### Description

This pull request changes a number of URLs to either fix 404s, or direct them to more relevant/up to date links. I also removed references to Bluemix since we've moved forward with netlify for the time being.

### Changelog

**Changed**

- Fixed broken 404s.
- Relative links to absolute links for external sites like npm.

**Removed**

- Bluemix manifest.yml files

### CODE REVIEW

File: .circleci/config.yml [hunk 1]
```diff
@@ -25,68 +25,6 @@ jobs:
             yarn ci-check
             yarn lerna run ci-check
 
-  deploy:
```
Comment (jeffchew) [inline .circleci/config.yml]: @photodow you will need to remove `deploy` under workflows too.

File: packages/patterns-react/.storybook/manager-head.html [hunk 2]
```diff
@@ -6,7 +6,7 @@
 <meta property="og:site_name" content="IBM.com Library Patterns">
 <meta property="og:description" content="A collection of IBM.com patterns implemented using React and Carbon Design System.">
 <meta property="og:image" content="https://media.github.ibm.com/user/525/files/59e3bfde-b990-11e7-87ef-072e89a87719">
-<meta property="og:url" content="https://ibmdotcomlibrary-patterns.mybluemix.net">
+<meta property="og:url" content="https://ibmdotcom-patterns.netlify.com">
```
Comment (jeffchew) [inline packages/patterns-react/.storybook/manager-head.html]: @photodow should be `ibmdotcom-patterns-react.netlify.com`

File: .circleci/config.yml (hunk 1 above)
Comment (photodow) [reply .circleci/config.yml]: Ok, awesome. I think I took care of it. Give it a second look and let me know.

File: packages/patterns-react/.storybook/manager-head.html (hunk 2 above)
Comment (photodow) [reply packages/patterns-react/.storybook/manager-head.html]: Thanks for catching!

**Pull request merged**